pkg/apis/kops/componentconfig.go

```diff
@@ -888,6 +888,23 @@ type NodeTerminationHandlerConfig struct {
 	CPURequest *resource.Quantity `json:"cpuRequest,omitempty"`
 }
 
+// NodeProblemDetector determines the node problem detector configuration.
+type NodeProblemDetectorConfig struct {
+	// Enabled enables the NodeProblemDetector.
+	// Default: false
+	Enabled *bool `json:"enabled,omitempty"`
+	// Image is the NodeProblemDetector docker container used
+	// Default: k8s.gcr.io/node-problem-detector/node-problem-detector:v0.8.7
+	Image string `json:"image,omitempty"`
```

Comment:
Any reason this is not *string type?
```suggestion
	Image string `json:"image,omitempty"`
```